dask/dataframe/core.py

```diff
@@ -4132,6 +4148,35 @@ def applymap(self, func, meta="__no_default__"):
     def round(self, decimals=0):
         return elemwise(M.round, self, decimals)
 
+    @derived_from(pd.DataFrame)
+    def mode(self, dropna=True, split_every=False):
+        mode_series_list = []
+        for col in self.columns:
+            mode_series = super(Series, self[col]).mode(
```

Comment:
```suggestion
            mode_series = self[col].mode(
```

Seems more future proof to call Series.mode, in case it's implementation changes in the future.

Reply:
This also may run into issues with duplicate column names, since the `self[col]` would return a DataFrame rather than a series. Perhaps safest to do positional slices or iterate over the columns.